src/app/lib/config/services/igbo.js

```diff
@@ -9,8 +9,10 @@ const igbo = {
   defaultImage: 'https://news.files.bbci.co.uk/ws/img/logos/og/igbo.png',
   defaultImageAltText: 'BBC News Ìgbò',
   brandSVG,
+  svgRatio: brandSVG.ratio,
```

Comment:
Do we need to add `svgRatio` to config?

Reply:
We actually don't have to - have removed svgRatio from config. svgRatio is now being retrieved from brandSvg